src/app/shared/form-controls/material-selection-list/material-selection-list.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from "@angular/core";
+import { FormGroup } from "@angular/forms";
+import { FormControlBase } from "../form-contol-base.model";
+
+@Component({
+  selector: "app-material-selection-list",
+  templateUrl: "./material-selection-list.component.html"
+})
+export class MaterialSelectionListComponent {
```

Comment:
Is this new component designed for our future use?

Reply:
Yes. In light of forthcoming additional filters, I was keen to also test my approach using selection lists.